includes/functions.php

```diff
@@ -284,3 +284,75 @@ function wepos_get_product_price( $product ) {
 
     return apply_filters( 'woocommerce_cart_product_subtotal', $product_subtotal, $product, $quantity, $this );
 }
+
+/**
+ * Get other templates (e.g. product attributes) passing attributes and including the file.
+ *
+ * @since 1.1.9
+ *
+ * @param mixed  $template_name
+ * @param array  $args          (Optional)
+ * @param string $template_path (Optional)
+ * @param string $default_path  (Optional)
+ *
+ * @return void
+ */
+function wepos_get_template( $template_name, $args = [], $template_path = '', $default_path = '' ) {
```

Comment:
I don't think we need these template helper functions as we don't require any templating in the free plugin, as well as in the pro plugin. For only displaying a missing dependency template, just use a `include/require_once` call.